lib/rubocop/cop/rspec/around_block.rb

```diff
@@ -31,7 +31,7 @@ class AroundBlock < Base
                          'or `%<arg>s.run`.'
 
         def_node_matcher :hook, <<-PATTERN
-          (block {(send nil? :around) (send nil? :around sym)} (args $...) ...)
+          (block (send nil? :around {sym | }) (args $...) ...)
```

Comment:
isn't there a syntax for optional node, e.g. `sym ?` (note the space is required in order not to turn `sym?` into a predicate

Reply:
Nice! There's no month without learning a new Node Pattern trick.